doc-content/enterprise-only/openshift/readwritemany-proc.adoc

```diff
@@ -0,0 +1,69 @@
+[id='readwritemany-proc']
+= Provisioning persistent volumes with ReadWriteMany access mode using NFS
+
+:rwm-authoring!:
+:rwm-bcmonitoring!:
+
+:rwm-gluster!:
+ifeval::["{context}"=="openshift-operator"]
+:rwm-authoring:
+:rwm-bcmonitoring:
+endif::[]
+ifeval::["{context}"=="openshift-authoring"]
+:rwm-authoring:
+:rwm-gluster:
+endif::[]
+ifeval::["{context}"=="openshift-authoring-managed"]
+:rwm-authoring:
+:rwm-gluster:
+endif::[]
+
+
+ifdef::rwm-authoring[]
+ifdef::PAM[]
+If you want to deploy 
+ifdef::rwm-bcmonitoring[]
+{CENTRAL} Monitoring, high-availability {CENTRAL},
+endif::rwm-bcmonitoring[]
+ifndef::rwm-bcmonitoring[]
+high-availability {CENTRAL}
+endif::rwm-bcmonitoring[]
+or any {KIE_SERVERS} that use the H2 database, which is the default setting for a non-high-availability authoring environment, your environment must provision persistent volumes with ReadWriteMany access mode.
+endif::PAM[]
+ifdef::DM[]
+If you want to deploy high-availability {CENTRAL}, your environment must provision persistent volumes with ReadWriteMany access mode.
+endif::DM[]
+
+ifdef::rwm-gluster[]
+[NOTE]
+====
+If you want to deploy a high-availability authoring environment, for optimal performance and reliability provision persistent volumes using GlusterFS. Configure the GlusterFS storage class as described in <<glusterfs-reconfig-proc>>.
+====
+endif::rwm-gluster[]
+endif::rwm-authoring[]
+
+:rwm-bcmonitoring!:
+ifeval::["{context}"=="openshift-freeform"]
+:rwm-bcmonitoring:
+endif::[]
+ifeval::["{context}"=="openshift-managed"]
+:rwm-bcmonitoring:
+endif::[]
+ifeval::["{context}"=="openshift-immutable"]
+:rwm-bcmonitoring:
+endif::[]
+ifdef::rwm-bcmonitoring[]
+If you want to deploy {CENTRAL} Monitoring, your environment must provision persistent volumes with ReadWriteMany access mode.
+endif::rwm-bcmonitoring[]
+
+If your configuration requires provisioning persistent volumes with ReadWriteMany access mode but your environment does not support such provisioning, use NFS to provision the volimes. Otherwise, skip this procedure.
+
+.Procedure
+
+Deploy an NFS server and provision the persistent volumes using NFS. For information about provisioning persistent volumes using NFS, see
+ifeval::["{context}"=="openshift-operator"]
+the "Persistent storage using NFS" section of the  https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/storage[OpenShift Container Platform 4.2 Storage] guide.
+endif::[]
+ifeval::["{context}"!="openshift-operator"]
+the "Persistent storage using NFS" section of the  https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/configuring_clusters/[OpenShift Container Platform 3.11 Installation and Configuration] guide.
```

Comment:
I see that this referenced guide is called "Configuring Clusters". Installation and configuration is the subtitle and seems to be just the category. Also, the doc title should be ital. And the section title I suppose ought to match the case used in the doc.

So this:

```
... the "Persistent Storage Using NFS" section of the  https.../[_Configuring Clusters_] guide.
```